schemas/2017-03-01-preview/Microsoft.Sql.json

```diff
@@ -326,36 +326,6 @@
       ],
       "description": "Microsoft.Sql/servers/databases/extendedAuditingSettings"
     },
-    "servers_databases_vulnerabilityAssessments": {
```

Comment:
>servers_databases_vulnerabilityAssessments [](start = 5, length = 42)

this needs to be removed from 2015-01-01/deploymentTemplate and 2014-04-01-preview/deploymentTemplate